.github/workflows/canary.yml

```diff
@@ -70,5 +70,5 @@ jobs:
       - env:
           TF_ACC: "1"
           TEST_CLOUD: ${{ matrix.cloud }}
-        run: go test -v -cover ./internal/provider/
```

Comment:
why's this different than the workflow timeout?

Reply:
The other sections can take longer I'd say. They are also timeouts covering different sets of work.